rest/src/main/kotlin/systems/ajax/englishstudytelegrambot/repository/WordCashableRepository.kt

```diff
@@ -0,0 +1,150 @@
+package systems.ajax.englishstudytelegrambot.repository
+
+import org.bson.types.ObjectId
+import org.slf4j.LoggerFactory
+import org.springframework.data.redis.core.ReactiveRedisTemplate
+import org.springframework.stereotype.Repository
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Mono
+import reactor.kotlin.core.publisher.switchIfEmpty
+import reactor.kotlin.core.publisher.toMono
+import systems.ajax.englishstudytelegrambot.entity.Word
+
+@Suppress("TooManyFunctions")
+interface WordCashableRepository {
```

Comment:
why do you need a separate cashable interface? can you reuse the existing interface?

Reply:
yeap, i thought about but wouldn't it to require use a lot of Qualifier?

Reply:
I guess managing qualifiers would be easier than managing two interfaces

Reply:
Done